omod/pom.xml

```diff
@@ -39,7 +39,7 @@
 		<dependency>
 			<groupId>org.bahmni.module</groupId>
 			<artifactId>teleconsultation-api</artifactId>
-			<version>1.0.0-SNAPSHOT</version>
+			<version>1.0.0</version>
```

Comment:
Why are you removing SNAPSHOT? This is not a released module. Or is this a dependency, and thats why?

Reply:
This commit was added by @rahu1ramesh 
I guess he removed it because of that module being used as dependency can revert the same